src/DevChatter.Bot.Core/Commands/TopBallersCommand.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using DevChatter.Bot.Core.Data;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Events.Args;
+using DevChatter.Bot.Core.Systems.Chat;
+
+namespace DevChatter.Bot.Core.Commands
+{
+    public class TopBallersCommand : BaseCommand
```

Comment:
How about a name change to be a `TopCommand`. "Ballers" is fun here, but we make sure this works for everyone. We can alias `!topballers` to trigger ours.